documentation/sign-in-canada.md

```diff
@@ -0,0 +1,36 @@
+# Sign In Canada
+
+## Local setup
+
+### Environment variables
+
+#### `api/.env`
+
+- Comment out block of variables 'for mock oauth testing'
+- Uncomment block of variables 'for SiC testing'
+- Change `OAUTH_ACR_VALUES=gckeymfa` to `OAUTH_ACR_VALUES=gckey`
+- Add values for `OAUTH_API_CLIENT_ID` and `OAUTH_API_CLIENT_SECRET` (these can be obtained from @tristan-orourke)
+
+#### `apps/web/.env`
+
+- Comment out block of variables 'Mock auth endpoint'
+- Uncomment block of variables 'SiC endpoint (will break e2e tests)'
+- Change `OAUTH_ACR_VALUES=gckeymfa` to `OAUTH_ACR_VALUES=gckey`
```

Comment:
This line is copy pasta, I think

Reply:
Updated in e50d7a4f659eeb698236b85bd400403fcd9477fc.

Reply:
Oops, I think you removed the line from the wrong block.

Reply:
😵 

1. fe4906a07c5b78613fd22ba71cf22680b5b612e6
2. 8df64e85b8a651bc137d637dfdaf8399761bd3da

Reply:
Nice.  :+1: